yoyodyne/data/datasets.py

```diff
@@ -120,7 +120,7 @@ def encode_target(self, symbols: List[str]) -> torch.Tensor:
         Returns:
             torch.Tensor.
         """
-        wrapped = symbols
+        wrapped = [s for s in symbols]
```

Comment:
Nasty bug. Let me suggest `wrapped = symbols.copy()` instead though.

Reply:
Good call, I forgot that `copy()` is faster.